causely/README.md

```diff
@@ -0,0 +1,40 @@
+# Causely - Assure Continuous Application Reliability using Causal AI
+
+## Overview
+
+The [Causal AI platform][1] automatically captures, represents and analyzes causality. It enables self-managed, resilient applications by bridging observability with automated orchestration. 
+
+Instead of flood of alerts, Causely tells you exactly what you need to know about all active and emerging application failures. With Causely, understand why a failure happened, what the root cause was, and how to fix it.
+
+Use Causely to:
+
+- **Assure continuous application reliability**: Guarantee that KPIs, SLAs, SLOs, and SLIs are consistently met to maintain application reliability and performance.
+- **Improve operational efficiency**: Reduce labor, data, and tooling costs by focusing on faster Mean Time to Repair (MTTR) and implementing efficient operational processes.
+- **Accelerate the delivery of features and innovations**: Enhance the speed and reliability of shipping new services and features to market, ensuring a competitive edge and meeting customer demands promptly.
+
+## Setup
+
+The Causely integration uses a [Datadog API key][2], which needs to be created by a Datadog admin.
```

Comment:
Can you provide the exact steps a user will take? What happens with the API key once created?

Reply:
the exact config is in the referenced documentation, but let me expand on what Causely will after it is configured